plugins/woocommerce/src/Blocks/BlockTypes/MiniCart.php

```diff
@@ -418,8 +420,12 @@ protected function get_markup( $attributes ) {
 		$wrapper_classes = sprintf( 'wc-block-mini-cart wp-block-woocommerce-mini-cart %s', $classes_styles['classes'] );
 		$wrapper_styles  = $classes_styles['styles'];
 
-		$icon_color          = array_key_exists( 'iconColor', $attributes ) ? esc_attr( $attributes['iconColor']['color'] ) : 'currentColor';
-		$product_count_color = array_key_exists( 'productCountColor', $attributes ) ? esc_attr( $attributes['productCountColor']['color'] ) : '';
+		$icon_color = isset( $attributes['iconColor'] ) && isset( $attributes['iconColor']['color'] )
+			? esc_attr( $attributes['iconColor']['color'] )
+			: 'currentColor';
+		$product_count_color = isset( $attributes['productCountColor'] ) && isset( $attributes['productCountColor']['color'] )
+			? esc_attr( $attributes['productCountColor']['color'] )
+			: '';
```

Comment:
Same here. Also, let's use the opportunity to use the correct capitalization for `currentcolor` (see https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword). 😉
```suggestion
		$icon_color          = $attributes['iconColor']['color'] ?? 'currentcolor';
		$product_count_color = $attributes['productCountColor']['color'] ?? '';
```